exemples/tls-auth/config/config.yaml

```diff
@@ -0,0 +1,62 @@
+common:
```

Comment:
is it really the minimun traefik config to have tls configured ?

Reply:
I'll need to update that, this file is not used because the container Crowdsec does'nt behave well.

There are some things that happen in the [entrypoint](https://github.com/crowdsecurity/crowdsec/blob/master/docker/docker_start.sh) of crowdsec container that does not like config binding

I used some Env variable
I could let a configuration as example for not docker case